storage/aptosdb/src/transaction_store/mod.rs

```diff
@@ -239,7 +241,9 @@ impl TransactionStore {
         end: Version,
         db_batch: &mut SchemaBatch,
     ) -> anyhow::Result<()> {
-        db_batch.delete_range::<TransactionInfoSchema>(&begin, &end)?;
+        for version in begin..end {
+            db_batch.delete::<TransactionInfoSchema>(&version)?;
```

Comment:
We did try restarting a node after pruner is activated, right? 
Reading the code, only the accumulator schema needs to be kept, for the `AccumulatorConsistencyProof`, but I just want  to make sure..